test/revise-spec.ts

```diff
@@ -28,75 +28,69 @@ describe('revise', () => {
     });
   });
 
-  describe('works with objects', () => {
+  describe('when objects passed in', () => {
     it('should return unchanged object if no change', () => {
-      const source = {foo: 1};
-      const revision = {foo: 1};
+      const source = {foo : 1};
```

Comment:
is this the recommended style?

seems bizarre to me

Reply:
No, I had an issue with my clang-formatter. See #19